### ISSUE TITLE
Mismatch between AWS ConfigurationSets and in the lint

### ISSUE THREAD

**Issue by Hedes-Productions** [aws-cloudformation/cfn-lint#2645]
### CloudFormation Lint Version

0.27.2

### What operating system are you using?

Mac

### Describe the bug

So today I ran into this error. The AWS cloud formation documentation [(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ses-configurationset.html)](aws) state that in AWS::SES::ConfigurationSet there is a property called SendingOptions. But when Im using lint it's giving me `[cfn-lint] E3002: Invalid Property Resources/BulkUploadClaimInviteConfigurationSet/Properties/SendingOptions
` error.

### Expected behavior

So I expect that this property also needed to be included in the lint.

### Reproduction template

`MyConfigurationSet:
    Type: AWS::SES::ConfigurationSet
    Properties:
      Name: !Ref MyConfigurationSet
      SendingOptions:
        SendingEnabled: true

  MyConfigurationSetDestination:
    Type: AWS::SES::ConfigurationSetEventDestination
    Properties:
      ConfigurationSetName: !Ref MyConfigurationSet
      EventDestination:
        Name: !Ref MyConfigurationSetDestination
        Enabled: true
        MatchingEventTypes:
          - send
          - open
          - reject
        CloudWatchDestination:
          DimensionConfigurations: 
            - DimensionName: "MyConfigurationSetMetrics"
              DimensionValueSource: "emailHeader"
              DefaultDimensionValue: !Ref "AWS::NoValue"`

**Comment by kddejong** [comment]
I believe this value exists in most recent versions of cfn-lint can you upgrade `cfn-lint`.  

If you are installing using pip you would run `pip install --upgrade cfn-lint`

If you are having issues can you let me know what region you are running in?

**Comment by Hedes-Productions** [comment]
Yeah, it got fixed. Actually when I installed the extension 0.27.2 version of cfn-lint is added. Thats why. So I used your hint and ran a `pip install --upgrade cfn-lint`. Now fixed. Yeah in new versions that got fixed.

> I believe this value exists in most recent versions of cfn-lint can you upgrade `cfn-lint`.
> 
> If you are installing using pip you would run `pip install --upgrade cfn-lint`
> 
> If you are having issues can you let me know what region you are running in?

Yeah, it got fixed. Actually when I installed the extension 0.27.2 version of cfn-lint is added. Thats why. So I used your hint and ran a `pip install --upgrade cfn-lint`. Now fixed. Yeah in new versions that got fixed.
